react app/src/App.css

```
@@ -1,42 +1,138 @@
+
+header p {
+  margin-top: 0;
+  color: #4b5563;
+}
+
+.controls {
+  background: #ffffff;
+  border-radius: 12px;
+  padding: 1rem;
+  border: 1px solid #e5e7eb;
+  box-shadow: 0 8px 20px rgba(15, 23, 42, 0.06);
```

Stylelint (17.2.0)

[error] Expected "0.06" to be "6%" (alpha-value-notation)

(alpha-value-notation)